### PR TITLE
[unittest] Archive unit test result

### PR DESCRIPTION
**Pull request by ohsewon**
archive unit test result using crontab, you can see it on'nnsuite.mooo.com'

Signed-off-by: sewon.oh <sewon.oh@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [*]Passed [ ]Failed []Skipped

### CODE REVIEW

File: .TAOS-CI/taos/auto-unittest.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    rm -rf ci/unittest_result/*
+else
+    mkdir -p ci/unittest_result
+fi
+for arch in $arch_type
+do
+    # Gbs build for unit test
+    # Unit test for ${arch}
+    gbs build -A ${arch} --overwrite --clean > temp.txt
+
```
Comment (myungjoo) [inline .TAOS-CI/taos/auto-unittest.sh]: trailing whitespace.

File: .TAOS-CI/taos/menu.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<html>
```
Comment (myungjoo) [inline .TAOS-CI/taos/menu.html]: Why this file is being installed in nnstreamer.git?
Isn't this supposed to be in TAOS-CI.git?

Comment (ohsewon) [reply .TAOS-CI/taos/menu.html]: Becaust this feature is only for nnstreamer.

File: .TAOS-CI/taos/auto-unittest.sh (hunk 1 above)
Comment (ohsewon) [reply .TAOS-CI/taos/auto-unittest.sh]: fixed.

File: .TAOS-CI/taos/auto-unittest.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+# Set-up environements
+dirpath="$( cd "$( dirname "$0")" && pwd )"
+build_root="${dirpath}/../../"
+arch_type="x86_64 armv7l"
+echo -e "[DEBUG] dirpath : $dirpath"
+echo -e "[DEBUG] build_root : $build_root"
+source ./common/api_collection.sh
+
+# Check dependency
+check_dependency gbs
```
Comment (leemgs) [inline .TAOS-CI/taos/auto-unittest.sh]: Please append rm, mv, and mkdir.

Comment (ohsewon) [reply .TAOS-CI/taos/auto-unittest.sh]: fixed

Comment (ohsewon) [reply .TAOS-CI/taos/auto-unittest.sh]: fixed

**Pull request merged**